.vscode/settings.json

```diff
@@ -2,9 +2,9 @@
     "restructuredtext.confPath": "${workspaceFolder}\\docs\\source",
     "python.formatting.blackArgs": [
         "--line-length",
-        "80",
+        "79",
         "--skip-string-normalization"
```

Comment:
Yeah, I ran into this too some time ago :)

Reply:
Yeah, I use 99 in my project specifically, which gives slightly more room for working with nested code and longer lines in general.

Reply:
I used to use 120 characters, but got used to 79, as this is still the standard in most projects - though sometimes it is inconvenient.